src/hooks/useSearchAPI/index.jsx

```diff
@@ -54,16 +78,18 @@ const useSearchAPI = (rootUrl, initialSearchParams={}, additionalParams={}) => {
   }
 
   useEffect(() => {
-    setPage(1)
-  }, [fulltext, selectedFacets, pageSize])
+    if (totalItems !== null) {
```

Comment:
This check just disables the resetting to page 1 on initial render. This bug was noticed when setting up pagination in the search params as it would always set the page to 1 when the dataset search page was loaded. `null` only exists before the first query has returned results.